definitions/output/reports/dynamic_publisher.js

```diff
@@ -0,0 +1,16 @@
+const configs = new reports.HTTPArchiveReports()
+const params = {
+  date: constants.currentMonth,
+  rankFilter: constants.devRankFilter
+}
```

Comment:
Query parameters.
I found only `date`.
Please list all the required and add the queries to test them with.